EnglishFilter/sections/antiadblock.txt

```diff
@@ -551,6 +551,8 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/77089
+@@||audiotag.info/js/adblockDetectorWithGA.js$domain=audiotag.info
```

Comment:
Remove it. Add rules below to the Annoyances - `antiadblock.txt`.
```
audiotag.info###adsclaim
audiotag.info##.dialog-fog
```

Reply:
Why?

Reply:
We can handle it in many ways, but if the simple solution works fine, we will add it.
Also this window can be closed, that's why it belongs to the Annoyances filter.